CHANGES.md

```diff
@@ -71,6 +71,17 @@ To be released.
         HashDigest<SHA256>?)` constructor instead.
  -  `IStore`, `IStateStore`, and `IKeyValueStore` interfaces now inherit
     `IDisposable`.  [[#1448], [#1474]]
+ -  Multiple changes for `IBlockPolicy<T>` interface.  [[#1485]]
+     -  `bool DoesTransactionFollowsPolicy(BlockChain<T>, Transaction<T>)`
+        changed to `TxPolicyViolationException? ValidateNextBlockTx(
```

Comment:
Instead of return exception, how about just throw exception and make return type to `void`? Also for `ValidateNextBlock`.

Reply:
I'm still on the fence about this decision. You are right in the sense that both methods should either throw designated exceptions or just return bool values. However, there are too many unknown factors at play, such as the precise responsibility of `IBlockPolicy` in particular, and the hierarchy of exceptions used by `IBlockPolicy` is not really finalized. I thought it wasn't worth the hassle and the risk to change them to throw exceptions at this moment. 😶